src/Parsers/Tree/Statements/UpdateNode.h

```diff
@@ -0,0 +1,38 @@
+#ifndef __UPDATE_NODE_H_
+#define __UPDATE_NODE_H_
+
+#include <set>
+#include <string>
+#include <vector>
+
+#include "StatementNode.h"
+
+namespace FQL
+{
+    class UpdateNode : public StatementNode
```

Comment:
Can you remove this from the PR as it will conflict with my `UpdateNode` class. 😅

Reply:
Sure